index.html

```diff
@@ -33,18 +37,19 @@
             <p id="discription-chose-lang">You're viewing YouTube in English (US). You can<span> change this preference below.</span> <span id="learn-more"><a target="blank" href="https://support.google.com/youtube/answer/87604?hl=en&visit_id=637000072769461721-1280205538&rd=1&co=GENIE.Platform%3DDesktop&oco=1">learn more</a></span></p>
         </div>
     </div>
+    <div class="container">
     <div id="grouping"> 
         <div class="video-player">
-        <iframe  width="640" height="360"
+        <iframe id="youtube-video"
             src="https://www.youtube.com/embed/tgbNymZ7vqY">
             </iframe>
             <h2>Contributing to Open Source Part I: The Easy Way </h2>
         </div>
             <div>
-        <img src="image/channel-icon.jpg" alt="channel-icon" class="subscribe-channel-button cusor"/> <p id="odin">The Odin Project</p>
```

Comment:
The image of the channel is not displaying, this is because it has the wrong extension, please change it to .JPG as it is in your image folder